lib/CoreBot.js

```diff
@@ -823,7 +823,7 @@ function Botkit(configuration) {
             // set up a once a second tick to process messages
             botkit.tickInterval = setInterval(function() {
                 botkit.tick();
-            }, 1000);
+            }, 1500);
```

Comment:
Turns out you can revert this as I don’t think it makes a difference. Basically the issue with the framework with Facebook is that if you send 2 or 3 messages, and one of them is an image for some reason it will come out of order. It’s something with facebooks queue on the backend. So i think what needs to happen is to wait in between messages before sending
--
Tom Kornblit

> On Jul 17, 2016, at 8:44 PM, Yagiz Nizipli <notifications@github.com> wrote:
> 
> In lib/CoreBot.js <https://github.com/howdyai/botkit/pull/317#discussion_r71093302>:
> 
> > @@ -823,7 +823,7 @@ function Botkit(configuration) {
> >              // set up a once a second tick to process messages
> >              botkit.tickInterval = setInterval(function() {
> >                  botkit.tick();
> > -            }, 1000);
> > +            }, 1500);
> Is there a particular reason to change the setInterval method to 1500 ms?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/howdyai/botkit/pull/317/files/809cf5d21ca7ca7ba5f4168cda8fb58ec0eb723c#r71093302>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AB7b-SlWA9071Oinid5i1DBWlSJPN4ZDks5qWsxvgaJpZM4JI56v>.
>